TypeCobol.LanguageServer/VsCodeProtocol/Text Document/VersionedTextDocumentIdentifier.cs

```diff
@@ -0,0 +1,20 @@
+﻿namespace TypeCobol.LanguageServer.VsCodeProtocol
+{
+    /// <summary>
+    /// An identifier to denote a specific version of a text document.
+    /// </summary>
+    public class VersionedTextDocumentIdentifier : TextDocumentIdentifier
+    {
+        /// <summary>
+        /// The version number of this document.
+        /// </summary>
+        public int? version { get; set; }
```

Comment:
Could you remove the `?` ? This is not nullable in LSP specs.

Reply:
Done